docs/gestor-dependencias.md

```diff
@@ -0,0 +1,14 @@
+# Gestor de dependencias
+En este fichero se va a analizar cuales son los posibles gestores de dependencias que hay para TypeScript y cuál y porqué se ha elegido para este proyecto.
```

Comment:
No, tienes que poner primero los criterios. Consulta errores frecuentes de las semana 8 a la 10, y presentaciones enlazadas relativas a ellos.

Reply:
Era solo una introducción y después están los criterios, pero lo quito si no es necesario.

Reply:
Creo que no lo entendéis. Por favor, mira los errores frecuentes de la semana 10. Sin no pones primero los requisitos y criterios, es que no estás siguiendo el procedimiento adecuado para elegirlo.